### DISCUSSION TITLE
Android CICD二维码下载如何推送给钉钉？

### DISCUSSION THREAD

**Discussion by HandsomeD123** [dqzboy/DevOps#3] [Q&A] [answered]
请问，我看你的Android  CICD截图里面有个下载二维码到钉钉机器人，这个是怎么实现的？

**Comment by dqzboy** [comment, marked as answer]
在这个是通过调取蒲公英的接口，上传到蒲公英之后，会有一个下载方式，获取这里的URL，截取二维码的部分，然后在推送给钉钉机器人
